CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.2
+
+* Internal code health improvements.
```

Comment:
Could say "Improved package description and added an example"

Reply:
Done